### ISSUE TITLE
Loading text popping up when a page is loading up 

### ISSUE THREAD

**Issue by DevEmmy** [fossology/FOSSologyUI#185]
### Description

When you load up a page, a "loading" text comes up which looks absurd, instead we could have a css loader or an animation.

#### Steps followed and expected result
Replace the text with a loader

**Comment by Shruti3004** [comment]
@DevEmmy Go ahead!

**Comment by krishna9304** [comment]
@DevEmmy are you still working on this?

**Comment by DevEmmy** [comment]
@krishna9304 Yes, But It might take time because I have a slight issue with my pc

**Comment by krishna9304** [comment]
> @krishna9304 Yes, But It might take time because I have a slight issue with my pc

So, can I take this up? @DevEmmy

**Comment by DevEmmy** [comment]
@krishna9304 
Let me get back to you by the end of today.

**Comment by DevEmmy** [comment]
@Shruti3004 I already created a pull request on that

**Comment by GMishx** [comment]
@DevEmmy Can you please add a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) to your PR so the issue gets linked with it?
